README.md

```diff
@@ -99,3 +99,17 @@ If we could keep the database (but move it to postgres) and also keep the api (p
 Possible mentor: @razzeee
 
 Difficulty: Hard
+
+## Replace Flatpak External Data Checker with Renovate
+
+[Flatpak External Data Checker](https://github.com/flathub-infra/flatpak-external-data-checker) is a tool maintained and operated by the Flathub team to help keep apps on Flathub up-to-date. Using annotations in the build manifest for each app, it monitors for new releases of each app and its bundled dependencies, and opens pull requests as needed to update them.
+
+While this tool has been useful in helping Flathub scale to its current size, it has limitations. It tends to open many overlapping pull requests when several modules of an application are updated at once; the build manifest annotations it consumes are somewhat ad-hoc and idiosyncratic; there are limits to what it can check and update; and and so on.
```

Comment:
Plus the checker can't be run on gitlab. (talked with a KDE person on the way back home and it seemed like this would help them)